### ISSUE TITLE
Brave slows down drastically if more than one window is open

### ISSUE THREAD

**Issue by pniedzwiedzinski** [brave/brave-browser#12780]
## Description 
Recently I updated brave from 1.12.112 to 1.15.76 (I don't know if an update is the case, but that information might be useful). And after a while, I noticed that sometimes the browser lags to the point that some sites are unusable (scrolling takes like 4s to start). My first thought was that I'm tight on ram and a lot of cards open - and that was true. But that also happens with one card, when I have more than one brave window open.

## Steps to Reproduce

   1. Launch more than one window of brave
   2. Check if everything runs smoothly or lags

## Reproduces how often: 
Not every time, seems a little random

## Brave version (brave://version info)

Brave | 1.15.76 Chromium: 86.0.4240.111 (Official Build) (64-bit)
-- | --
Revision | b8c36128a06ebad76af51591bfec980224db5522-refs/branch-heads/4240@{#1290}
OS | Linux
JavaScript | V8 8.6.395.17
Flash | (Disabled)
User Agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36
Command Line | /nix/store/082i32da729lqfv71y1fja37ybwyfznh-brave-1.15.76/opt/brave.com/brave/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --enable-features=AutoupgradeMixedContent,PassiveMixedContentWarning,ReducedReferrerGranularity,PasswordImport,OmniboxContextMenuShowFullUrls,LegacyTLSEnforced,DnsOverHttps,MixedContentSiteSetting,WebUIDarkMode,PrefetchPrivacyChanges --disable-features=TextFragmentAnchor,AutofillEnableAccountWalletStorage,AllowPopupsDuringPageUnload,AutofillServerCommunication,PasswordCheck,NotificationTriggers,PrivacySettingsRedesign,SafeBrowsingEnhancedProtection,TabHoverCards,SmsReceiver,VideoPlaybackQuality --flag-switches-begin --flag-switches-end
Executable Path | /nix/store/082i32da729lqfv71y1fja37ybwyfznh-brave-1.15.76/opt/brave.com/brave/brave
Profile Path | /home/pn/.config/BraveSoftware/Brave-Browser/Default

## Version/Channel Information:
I tested it with v1.12.112 and it works fine, but if I open two windows of v1.15.76, then the v1.12.112 window also gets unresponsive.

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by rebron** [comment]
@pniedzwiedzinski Can you update to the latest 1.17.73 and let us know if you're still having a slow down with our latest version.

**Comment by pniedzwiedzinski** [comment]
Hi, it looks like the new release fixes it, whatever that was. Thanks,
this can be closed now.

**Comment by kratosok** [comment]
I too seem to have this problem in the current beta as well. I've got a multimonitor setup, the performance seems normal if the two Brave windows are on different screens. Video playback(youtube) in one of the windows seems to make the situation worse.

OS: Fedora silverblue(currently based on fedora 33)
Desktop: Gnome/Wayland

chrome://version-

```
Brave	1.19.46 Chromium: 87.0.4280.67 (Official Build) beta (64-bit)
Revision	0e5d92df40086cf0050c00f87b11da1b14580930-refs/branch-heads/4280@{#1441}
OS	Linux
JavaScript	V8 8.7.220.25
Flash	(Disabled)
User Agent	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.67 Safari/537.36
Command Line	/usr/lib/opt/brave.com/brave-beta/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=AutoupgradeMixedContent,WebUIDarkMode,DnsOverHttps,LegacyTLSEnforced,PrefetchPrivacyChanges,PasswordImport,ReducedReferrerGranularity --disable-features=AutofillServerCommunication,TabHoverCards,TextFragmentAnchor,NotificationTriggers,SmsReceiver,PrivacySettingsRedesign,VideoPlaybackQuality,AllowPopupsDuringPageUnload,AutofillEnableAccountWalletStorage,PasswordCheck,NetworkTimeServiceQuerying,IdleDetection,SignedExchangeSubresourcePrefetch,SafeBrowsingEnhancedProtection --flag-switches-begin --flag-switches-end
Executable Path	/usr/lib/opt/brave.com/brave-beta/brave
Profile Path	/var/home/user/.config/BraveSoftware/Brave-Browser-Beta/Default
```

**Comment by VersBinarii** [comment]
Seeing similar issue on multimonitor setup. Its extremely slow with 2 windows open. No matter if they on the same display or different ones.

I'm running Arch Linux with Sway.

```
Brave	1.17.75 Chromium: 87.0.4280.88 (Official Build) unknown (64-bit)
Revision	89e2380a3e36c3464b5dd1302349b1382549290d-refs/branch-heads/4280@{#1761}
OS	Linux
JavaScript	V8 8.7.220.29
Flash	(Disabled)
User Agent	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Command Line	/usr/lib/brave-bin/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --enable-features=AutoupgradeMixedContent,DnsOverHttps,LegacyTLSEnforced,PasswordImport,ReducedReferrerGranularity,PrefetchPrivacyChanges,WebUIDarkMode --disable-features=AutofillEnableAccountWalletStorage,PasswordCheck,SmsReceiver,NotificationTriggers,TextFragmentAnchor,VideoPlaybackQuality,AllowPopupsDuringPageUnload,IdleDetection,AutofillServerCommunication,PrivacySettingsRedesign,TabHoverCards,SignedExchangeSubresourcePrefetch,SafeBrowsingEnhancedProtection --flag-switches-begin --flag-switches-end
Executable Path	/usr/lib/brave-bin/brave
Profile Path	/home/user/.config/BraveSoftware/Brave-Browser/Default
```

**Comment by 71walceli** [comment]
I started to have this from a few months ago. I downgraded it, downgraded some other X packages as I thought they were the root cause, and even rebooted to previous kewrnels, none of which actually worked. Brave was putting out 1 or 2 frames, sometimes, it was completely frozen. Yet is seemed responsive. 

What seemed to work is disabling HW acceleration from settings... since then, it's been OK.

Hope this helps...

**Comment by komandar** [comment]
@bsclifton 
The problem has been going on for over 4 years now.
Unfortunately, it does not seem to be taken care of. Is there a reason for this?
The fact that I quickly found this issue and others (e.g. https://github.com/brave/brave-browser/issues/16453) means that this cannot be an isolated problem.

How is the problem planned to be dealt with?

**Comment by Delgan** [comment]
@komandar The related issues being closed, I assume the Brave developers are not aware this is still an actual problem. Surely, we should open a new ticket with additional details.

Personally, I systematically experience intermittent freezes when opening two Brave instances playing video media in parallel. Disabling hardware acceleration seems to solve the problem, but it's obviously not a desirable solution. I'm using Linux 6.12.

**Comment by 71walceli** [comment]
> I started to have this from a few months ago. I downgraded it, downgraded some other X packages as I thought they were the root cause, and even rebooted to previous kewrnels, none of which actually worked. Brave was putting out 1 or 2 frames, sometimes, it was completely frozen. Yet is seemed responsive.
> 
> What seemed to work is disabling HW acceleration from settings... since then, it's been OK.

In my experience, this hasn't happened anymore. Didn't need to touch anything so far. I don't think I can find exact setting I toggled back then, but it has to do probably with video/grapthics/hardware acceleration. 

This thing can have so many causes for all we know.

**Comment by 71walceli** [comment]
> [@komandar](https://github.com/komandar) The related issues being closed, I assume the Brave developers are not aware this is still an actual problem. Surely, we should open a new ticket with additional details.
> 
> Personally, I systematically experience intermittent freezes when opening two Brave instances playing video media in parallel. Disabling hardware acceleration seems to solve the problem, but it's obviously not a desirable solution. I'm using Linux 6.12.

It could be drivers, or too much CPU usage per video/tab

**Comment by Cifer9516** [comment]
Still experience this issue on Version 1.77.97 Chromium: 135.0.7049.84 (Official Build) (64-bit)

**Comment by alvin883** [comment]
Is anyone have a workaround for this? it's still happen on Version 1.78.94 Chromium: 136.0.7103.60 (Official Build) (64-bit), it drives me crazy if I had to open two window at the same time
